typescript/sdk/src/deploy/HyperlaneDeployer.ts

```diff
@@ -258,6 +261,20 @@ export abstract class HyperlaneDeployer<
     );
   }
 
+  // prettier-ignore
```

Comment:
why?

Reply:
I was trying to get it to ignore the line below so I could remove the @ts-ignore. Need help from @yorhodes here